cli-tests/src/upload.rs

```diff
@@ -210,28 +210,40 @@ async fn upload_bundle() {
 async fn upload_bundle_using_bep() {
     let temp_dir = tempdir().unwrap();
     generate_mock_git_repo(&temp_dir);
-    generate_mock_bazel_bep(&temp_dir);
+    let bep_path = generate_mock_bazel_bep(&temp_dir);
 
     let state = MockServerBuilder::new().spawn_mock_server().await;
 
     let assert = CommandBuilder::upload(temp_dir.path(), state.host.clone())
-        .bazel_bep_path("./bep.json")
+        .bazel_bep_path(bep_path.to_str().unwrap())
         .command()
         .assert()
-        .failure();
+        .success();
```

Comment:
It seems we weren't generating BEP files correctly and so the upload failed when it shouldn't have